### PR TITLE
Add theme to app

### PR DESCRIPTION
**Pull request by nedSaf**
#101

### CODE REVIEW

**Comment by nedSaf** [comment]
Update: Dashboard
localhost_9000__code_795748454d8040695a05__

**Comment by nedSaf** [comment]
Update: 

Dashboard:
localhost_9000__code_795748454d8040695a05__homepage

Repos:
localhost_9000__code_795748454d8040695a05__repos

**Comment by nedSaf** [comment]
Update: Builds list
localhost_9000___builds

File: client/app/views/dashboard/screenshots/screenshots.html [hunk 1]
```diff
@@ -52,39 +75,55 @@
...
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <div ng-style="{'height' : screenshot.maxHeight[imageStyle] }">
+
+              <img class="diff-image" ng-show="showDiff" ng-src="{{ imageStyle == 'self' ? (screenshot.diff.self + '?access_token=' + accessToken) : (screenshot.diff.styles[imageStyle]['url'] + '&access_token=' + accessToken)}}" />
+
+              <before-after-image-slider ng-hide="showDiff"
+                                         baseline-img-src="{{ imageStyle == 'self' ? (screenshot.baseline.self + '?access_token=' + accessToken) : (screenshot.baseline.styles[imageStyle]['url'] + '&access_token=' + accessToken)}}"
```
Comment (amitaibu) [inline client/app/views/dashboard/screenshots/screenshots.html]: can this be pulled back - so the line becomes shorter

Comment (amitaibu) [inline client/app/views/dashboard/screenshots/screenshots.html]: sorry, wrong PR :)

**Comment by nedSaf** [comment]
Update: Finished repositories page:
localhost_9000___repos

**Comment by nedSaf** [comment]
Screenshots page is still WIP.

**Comment by nedSaf** [comment]
Browser stack:
localhost_9000__code_66b34dbc26f25cfa1f26__my-account

Screenshots:
screenshot

**Comment by nedSaf** [comment]
@amitaibu Ready for review.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**